playbooks/de-populate-consul.yaml

```diff
@@ -272,6 +272,11 @@
         - {key: "configs/{{environment_name}}/nginx/ssl/key", value: "{{ de.http_server.ssl.cert_key }}"}
         - {key: "configs/{{environment_name}}/nginx/insecure-redirects", value: "___ANSIBLE_IS_TERRIBLE___{% if de.http_server.ssl.insecure_redirects is defined and de.http_server.ssl.insecure_redirects is not none %}{{ de.http_server.ssl.insecure_redirects | to_json }}{% endif %}"}
         - {key: "configs/{{environment_name}}/nginx/redirects", value: "___ANSIBLE_IS_TERRIBLE___{% if de.http_server.ssl.redirects is defined and de.http_server.ssl.redirects is not none %}{{ de.http_server.ssl.redirects | to_json }}{% endif %}"}
+        - {key: "configs/{{environment_name}}/de/intercom-enabled", value: "{{ intercom_enabled }}"}
+        - {key: "configs/{{environment_name}}/de/intercom-app-id", value: "{{ app_id }}"}
+        - {key: "configs/{{environment_name}}/de/intercom-company-id", value: "{{ company_id }}"}
+        - {key: "configs/{{environment_name}}/de/intercom-company-name", value: "{{ company_name }}"}
```

Comment:
Could these moved up with the other `/de/` settings?

On the other hand, we talked about potentially submitting events to intercom from our services in the future, so maybe these should be their own `/intercom/` settings?